website/templates/project/nodes_privacy.mako

```diff
@@ -0,0 +1,111 @@
+                                <div class="spinner-loading-wrapper">
+                                    <div class="logo-spin logo-md"></div>
+                                    <p class="m-t-sm fg-load-message"> Loading projects and components...  </p>
+                                </div>
+                            </div>
+                            <div class="help-block" style="padding-left: 15px">
+                                <p id="configureNotificationsMessage"></p>
+                            </div>
+                        </div>
+                </div>
+                <!-- end select projects page -->
+
+                <!-- addon and projects changed warning page -->
+
+                <div data-bind="if: page() == 'addon'">
+                    <div data-bind="visible: nodesChangedPublic().length > 0">
+                        <div class="panel panel-default">
+                            <div class="panel-heading clearfix">
+                                <h3 class="panel-title" data-bind="html:message()['nodesPublic']"></h3>
+                            </div>
+                            <div class="panel-body">
+                                <ul data-bind="foreach: { data: nodesChangedPublic, as: 'item' }">
+                                    <li>
+                                        <h4 class="f-w-lg" data-bind="text: item"></h4>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div data-bind="visible: nodesChangedPrivate().length > 0">
+                        <div class="panel panel-default">
+                            <div class="panel-heading clearfix">
+                                <h3 class="panel-title" data-bind="html:message()['nodesPrivate']"></h3>
+                            </div>
+                            <div class="panel-body">
+                                <ul data-bind="foreach: { data: nodesChangedPrivate, as: 'item' }">
+                                    <li>
+                                        <h4 class="f-w-lg" data-bind="text: item"></h4>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <!-- end projects changed warning page -->
+
+                </div>
+            </div><!-- end modal-body -->
+
+            <div class="modal-footer">
+
+                <a href="#" class="btn btn-default" data-bind="click: clear" data-dismiss="modal">Cancel</a>
+
+                <span data-bind="if: page() == 'warning'">
+                    <a class="btn btn-primary" data-bind="click:selectProjects">Next</a>
+                </span>
+
+                <span data-bind="if: page() == 'select'">
+                    <a class="btn btn-primary" data-bind="click:addonWarning">Next</a>
```

Comment:
Minor: Rather than having two "Next" buttons, you could do

```js
 <a class="btn btn-primary" data-bind="click: page() === 'warning' ? selectProjects : addonWarning">Next</a>
```

Alternatively, you could add some simple logic to the view model

```js
self.handleClickNext = ko.computed(function() { self.page() === 'warning' ? self.selectProjects : self.addonWarning;})
```

```html
 <a class="btn btn-primary" data-bind="click: handleClickNext()">Next</a>
```